documentation/guidance/gsuite/collaborative-inbox-forwarding.md

```diff
@@ -16,18 +16,18 @@ If you don't have [two-factor authentication enabled](https://www.google.com/lan
 | Email address | `info@helpfulengineering.org` | Address of the shared inbox |
 | Treat as an alias | ✓ | Send messages on behalf of the shared inbox account |
 
-### 4. Configure the acount<sup>2</sup>
+### 4. Configure the acount<sup>1</sup>
```

Comment:
Is there a small typo here? Should `acount` be `account`?

Reply:
```suggestion
### 4. Configure the account<sup>1</sup>
```
Certainly it is: it beats the semicolon as hide-and-seek champion.
Thank you very much, @skuenzli!